cmd/rofl/build/build.go

```diff
@@ -333,9 +327,14 @@ func init() {
 	buildFlags := flag.NewFlagSet("", flag.ContinueOnError)
 	buildFlags.BoolVar(&offline, "offline", false, "do not perform any operations requiring network access")
 	buildFlags.StringVar(&outputFn, "output", "", "output bundle filename")
-	buildFlags.BoolVar(&doUpdate, "update-manifest", false, "automatically update the manifest")
+	buildFlags.BoolVar(&noUpdate, "no-update-manifest", false, "do not update the manifest")
 	buildFlags.BoolVar(&doVerify, "verify", false, "verify build against manifest and on-chain state")
 	buildFlags.StringVar(&deploymentName, "deployment", buildRofl.DefaultDeploymentName, "deployment name")
 
+	// TODO: Remove when all the examples, demos and docs are updated.
+	dummy := true
+	buildFlags.BoolVar(&dummy, "update-manifest", true, "not update the manifest")
+	_ = buildFlags.MarkDeprecated("update-manifest", "the app manifest is updated by default. Pass --no-update-manifest to prevent updating it.")
```

Comment:
```suggestion
	_ = buildFlags.MarkDeprecated("update-manifest", "The app manifest is now updated by default. Pass --no-update-manifest to prevent updating it.")
```

Reply:
Actually, Cobra formats a message for you ending with comma, so the message should start with lower case:

```
Flag --update-manifest has been deprecated, the app manifest is now updated by default. Pass --no-update-manifest to prevent updating it.
```